src/components/AffectedOfferingForm.vue

```diff
@@ -14,7 +14,13 @@ import {
 } from '@/types/zodAffect';
 
 const { width: screenWidth } = useWindowSize();
-
+const resolutionOptions = computed(() => {
+  return {
+    AFFECTED: affectResolutions,
+    NEW: [''],
+    NOTAFFECTED: [''],
+  }[modelValue.value?.affectedness as string] || [''];
```

Comment:
What would be the displayed option in case `affectedness` is null?

Reply:
`''`